qiskit_experiments/library/calibration/drag.py

```diff
@@ -109,6 +113,19 @@ def _default_experiment_options(cls) -> Options:
 
         return options
 
+    @classmethod
+    def _default_calibration_options(cls) -> Options:
+        """Default calibration options for the experiment.
+
+        Calibration Options:
+            cal_parameter_name (str): The name of the DRAG parameter in the schedule stored in
+                the calibrations instance. The default value is "β".
+        """
+        options = super()._default_calibration_options()
+        options.cal_parameter_name = "β"
```

Comment:
The use here seems to suggest that these can be optional init kwargs for the base experiment, rather than separate options.

Reply:
See here: 5c2d9c1